Userland/SerenityBox/main.cpp

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Format.h>
+#include <AK/StringView.h>
+#include <AK/Vector.h>
+#include <LibC/unistd.h>
+#include <LibMain/Main.h>
+#include <string.h>
+#include <time.h>
+
+ErrorOr<int> serenity_main(Main::Arguments)
+{
+    dbgln("Hello SerenityOS Box");
```

Comment:
What is actually the plan here on how you will implement this feature-wise? You compared this to busybox, so I assume that it will be a bunch of command line tools bundled into one static binary. Are we going to import source from `Userland/Utilities/*`? Because otherwise I'm not looking forwards toward having multiple implementations around.

Reply:
The idea is to actually take the code from what is already present, yes.